src/main/java/RequestVerifier.java

```diff
@@ -0,0 +1,67 @@
+package com.td.HTTPServer;
+
+public class RequestVerifier implements HTTPVerifierInterface {
+
+  public boolean isValidHTTP(String request) {
+    String request_array[] = request.split("\\s+");
+    return isFirstParamaterValid(request_array[0]) &&
+           isSecondParamaterValid(request_array[1]) &&
+           isThirdParameterValid(request_array[2]);
+  }
+
+  private boolean isFirstParamaterValid(String first) {
+    return isRequestGet(first) ||
+           isRequestHead(first) ||
+           isRequestPost(first) ||
+           isRequestPut(first) ||
+           isRequestDelete(first) ||
+           isRequestTrace(first) ||
+           isRequestOptions(first) ||
+           isRequestConnect(first) ||
+           isRequestPatch(first);
+  }
+
+  private boolean isSecondParamaterValid(String second) {
+    return second.substring(0, 1).equals("/");
+  }
+
+  private boolean isThirdParameterValid(String third) {
```

Comment:
`third` and `second` are pretty arbitrary variable/argument names, and kinda reveal an assumption about the format of an HTTP request line. I would recommend first identifying what each of these components are and using those names. Furthermore, rather than dealing with a `String` throughout all this work, you may want to wrap the whole request in its own object...